examples/run_field_dep_justMw.py

```diff
@@ -65,7 +64,7 @@
 min_dBm_step = 0.5
 for x in range(len(powers)):
     dB_settings = (
-        round(10 * (np.log10(powers[x]) + 3.0) / min_dBm_step) * min_dBm_step
+        round(10 * (log10(powers[x]) + 3.0) / min_dBm_step) * min_dBm_step
```

Comment:
Why do the import statement differently and then change this? This is just an unnecessary change.